libs/barista-components/chart/src/chart-module.ts

```diff
@@ -40,6 +41,8 @@ const COMPONENTS = [
   DtChartTimestamp,
   DtChartTooltip,
   DtChartSelectionAreaAction,
+  DtChartFocusAnchor,
```

Comment:
Do we really want to export this from the module? isn't this just used internally?

Reply:
yeah. we can make this internal